cmd/process-agent/collector.go

```diff
@@ -360,14 +368,16 @@ func (l *Collector) run(exit chan struct{}) error {
 				statsd.Client.Gauge("datadog.process.agent", 1, tags, 1) //nolint:errcheck
 			case <-queueSizeTicker.C:
 				updateQueueStats(&queueStats{
-					processQueueSize:    l.processResults.Len(),
-					rtProcessQueueSize:  l.rtProcessResults.Len(),
-					eventQueueSize:      l.eventResults.Len(),
-					podQueueSize:        l.podResults.Len(),
-					processQueueBytes:   l.processResults.Weight(),
-					rtProcessQueueBytes: l.rtProcessResults.Weight(),
-					eventQueueBytes:     l.eventResults.Weight(),
-					podQueueBytes:       l.podResults.Weight(),
+					processQueueSize:      l.processResults.Len(),
+					rtProcessQueueSize:    l.rtProcessResults.Len(),
+					connectionsQueueSize:  l.connectionsResults.Len(),
+					eventQueueSize:        l.eventResults.Len(),
+					podQueueSize:          l.podResults.Len(),
+					processQueueBytes:     l.processResults.Weight(),
+					rtProcessQueueBytes:   l.rtProcessResults.Weight(),
+					connectionsQueueBytes: l.connectionsResults.Weight(),
+					eventQueueBytes:       l.eventResults.Weight(),
+					podQueueBytes:         l.podResults.Weight(),
 				})
 			case <-queueLogTicker.C:
 				l.logQueuesSize()
```

Comment:
Does this function need to be updated as well? Not sure how we decide which queues to log or not

Reply:
Good catch, I missed this spot.